.ruby-version

```diff
@@ -1 +1 @@
-2.3.7
+2.3.5
```

Comment:
@maikelmclauflin is there a reason why you bumped this down, or can we keep it at 2.3.7?